content/en/examples/application/nginx-app-list.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
```

Comment:
please remove this example.
Examples under the `content/en/examples` subdirectory are subject to syntax check using a validation logic. The `kind: List` will definitely fail because it is not an API server thing. It is only a client side convenience invented by `kubectl`.

Also, examples under the `content/en/examples` subdirectory are meant to be referenced from the text somewhere. They are not designed to be exposed directly to users. In other words, if an example is not referenced anywhere (using the `codenew` shortcode), it is invisible to all.

Reply:
Yes, I removed the file that was referencing the file earlier today and have now removed the file itself. All the substantive content of this PR is now in the api-concepts.md file. Please review the remaining content.